packages/react-ui/components/TokenInput/TokenInput.tsx

```diff
@@ -51,13 +51,28 @@ export enum TokenInputType {
 export type TokenInputMenuAlign = 'left' | 'cursor';
 
 export interface TokenInputProps<T> extends CommonProps {
+  /**
+   * Выбранные токены, которые будет отображаться в поле вводе
```

Comment:
Сори, орфографию мне лучше не доверять 🤪

_Предложение можно влить прямо из ПРа. Гитхаб позволит ввести заголовок коммита а дальше сам все сделает._

```suggestion
   * Выбранные токены, которые будут отображаться в поле ввода
```